docs/guides/how_to_guides/configuring_data_contexts/how_to_instantiate_a_data_context_on_a_databricks_spark_cluster.rst

```diff
@@ -27,49 +27,93 @@ This how-to-guide assumes that you are using a Databricks Notebook, and using th
+
+    .. note::
+       If you are using DBFS for your stores, make sure to set the ``root_directory`` of FilesystemStoreBackendDefaults to ``/dbfs/`` or ``/dbfs/FileStore/`` to make sure you are writing to DBFS and not the Spark driver node filesystem. If you have mounted another file store (e.g. s3 bucket) to use instead of DBFS, you can use that path here instead.
+
+    .. code-block:: python
+        :linenos:
+
+        from great_expectations.data_context.types.base import DataContextConfig
+        from great_expectations.data_context import BaseDataContext
+
+        data_context_config = DataContextConfig(
+            datasources={<set_your_datasources_here>},
+            store_backend_defaults=FilesystemStoreBackendDefaults(root_directory="/dbfs/FileStore/"),
+        )
+        context = BaseDataContext(project_config=data_context_config)
+
+    You can have more fine-grained control over where your stores are located by passing the ``stores`` parameter to DataContextConfig as in the following example.
+
+    .. content-tabs::
+
+        .. tab-container:: tab0
+            :title: Docs for Legacy Checkpoints (<=0.13.7)
+
+            .. code-block:: python
+                :linenos:
+
+                data_context_config = DataContextConfig(
+                    datasources={<set_your_datasources_here>},
+                    stores={
+                        "insert_your_custom_expectations_store_name_here": {
+                            "class_name": "ExpectationsStore",
+                            "store_backend": {
+                                "class_name": "TupleFilesystemStoreBackend",
+                                "base_directory": "/dbfs/FileStore/path_to_your_expectations_store/",
+                            },
+                        },
+                        "insert_your_custom_validations_store_name_here": {
+                            "class_name": "ValidationsStore",
+                            "store_backend": {
+                                "class_name": "TupleFilesystemStoreBackend",
+                                "base_directory": "/dbfs/FileStore/path_to_your_validations_store/",
+                            },
+                        },
+                        "insert_your_custom_evaluation_parameter_store_name_here": {
+                            "class_name": "EvaluationParameterStore"
+                        },
+                    },
+                    store_backend_defaults=FilesystemStoreBackendDefaults(root_directory="/dbfs/FileStore/"),
+                    )
+
+        .. tab-container:: tab1
+            :title: Docs for Class-Based Checkpoints (>=0.13.8)
+
+            .. code-block:: python
+                :linenos:
+
+                data_context_config = DataContextConfig(
+                    datasources={<set_your_datasources_here>},
```

Comment:
This is a little nit-picky, but is it possible to add a link or short description for what should be done with  `<set_your_datasources_here>`?

Reply:
It's also likely that it was me who wrote this part in the first place.. :)

Reply:
Good call! 
FYI on my thinking here: I was thinking of leaving it as-is to make these docs a bit more DRY, e.g. the link to the creating a context without a yml file shows examples of setting datasources in code. But I think it makes sense to delight the user by having all the info on one page, especially since the example at the end of this doc shows a datasource for 0.13+ so I'm not even being internally consistent 😂 . tl;dr: I'll add datasource examples.

Reply:
I added more examples, see commit 54eabbf